ethstats/ethstats.go

```diff
@@ -244,20 +262,20 @@ func (s *Service) readLoop(conn *websocket.Conn, in *json.Decoder) {
 			// Make sure the request is valid and doesn't crash us
 			request, ok := msg["emit"][1].(map[string]interface{})
 			if !ok {
-				log.Warn("Invalid history request", "msg", msg["emit"][1])
-				return
+				log.Warn("Invalid stats history request", "msg", msg["emit"][1])
```

Comment:
Question, if I understand it correct ethstats asks for the latest 3k blocks for a new connection and if this node hasn't got them it could close the connection in the old situation and now we basically ignore it. Since we keep the connection open could this cause issues in the ethstats page where it would have registered some kind of callback that is never going to fire?

Also, maybe we can drop the log level from warn to debug since is seems to be something that is done intentionally and not really something harmful.